WBooks/Screens/Library/LibraryViewController.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  ViewController.swift
+//  WBooks
+//
+//  Created by Guido Marucci Blas on 4/3/16.
```

Comment:
o.O

Reply:
I modified the default .swift that came with the project. Should I override the creation / author info?

Reply:
ohh I see haha, I thought you created the file.